From: Ben Hutchings <ben@decadent.org.uk>
Date: Fri, 15 Feb 2013 02:35:13 +0000
Subject: x86/PCI: Use _CRS by default on VMware
Bug-Debian: http://bugs.debian.org/700544

VMware supports memory hotplug and we need to avoid mapping PCI devices
in the address ranges where it will use for this.

This change is effectively included in commit
7bc5e3f2be32ae6fb0c74cd0f707f986b3a01a26 upstream, but I don't
want to include that in squeeze in case it causes a regression.

--- a/arch/x86/pci/acpi.c
+++ b/arch/x86/pci/acpi.c
@@ -32,6 +32,14 @@
 			DMI_MATCH(DMI_PRODUCT_NAME, "x3800"),
 		},
 	},
+	/* http://bugs.debian.org/700544 */
+	{
+		.callback = set_use_crs,
+		.ident = "VMware",
+		.matches = {
+			DMI_MATCH(DMI_PRODUCT_NAME, "VMware Virtual Platform"),
+		},
+	},
 	{}
 };
 
